### PR TITLE
Roll GLSLang and SPIRV-Tools

### PR DESCRIPTION
**Pull request by dj2**
This CL rolls the GLSLang and SPIRV-Tools dependencies to tip-of-tree
and updates the tests as required.

### CODE REVIEW

File: reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-inverted.asm.invalid.frag [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#version 450
```
Comment (HansKristian-Work) [inline reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-inverted.asm.invalid.frag]: The old non-invalid file lingers.

File: reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-non-inverted.asm.invalid.frag [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#version 450
```
Comment (HansKristian-Work) [inline reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-non-inverted.asm.invalid.frag]: The old non-invalid reference file lingers.

File: reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-inverted.asm.invalid.frag (hunk 1 above)
Comment (dj2) [reply reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-inverted.asm.invalid.frag]: Fixed.

File: reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-non-inverted.asm.invalid.frag (hunk 2 above)
Comment (dj2) [reply reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-non-inverted.asm.invalid.frag]: Fixed.

**Pull request merged**